Mocks/Http/MockedHttpClientBuilderExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿// ==++==
+//
+//   Copyright (c) Shadowsoft Corporation.  All rights reserved.
+//
+// ==--==
+
+using Microsoft.Extensions.Logging;
+
+namespace Yextly.Testing.Mocks.Http
+{
+    /// <summary>
+    /// Provides extension methods to create <see cref="HttpClient"/> instances.
+    /// </summary>
+    public static class MockedHttpClientBuilderExtensions
+    {
+        private static readonly Lazy<LeakingContainer> _leakingContainer = new(() => new LeakingContainer(), LazyThreadSafetyMode.ExecutionAndPublication);

```

Comment:
I don't quite get what this container used in one path (the DI calls `CreateHandler` and does not reach this point)

Reply:
Thanks